src/components/page/Page.js

```diff
@@ -0,0 +1,83 @@
+import React, { useMemo } from "react";
+import BranchTable from "./BranchTable";
+import PrTable from "./PrTable";
+import GithubQuery from "../../lib/query";
+import { useGithubApi } from "../../lib/useGithubApi";
+
+export const Page = ({ repos }) => {
+  const { authed, error, fetchData } = useGithubApi(async () => {});
+
+  const baseQuery = useMemo(
```

Comment:
so this is intended to never rerun unless the component is remounted?

Reply:
this isn't new code, this is just a component I extracted